### PR TITLE
mongoDB-api-liza

### PR DESCRIPTION
**Pull request by liza-minican**
Hey!

Merry Christmas and happy New Year! thanks for all the support and knowledge you give us!! 

Here is my project live:
https://mongo-api-deployment-liza.herokuapp.com

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -3,36 +3,130 @@ import bodyParser from 'body-parser'
...
+}
 })
 
+app.get('/', (req, res) => {
+    res.send('Hello, welcome to this music records api built with MongoDB')
+  })
+
+//Find all records and query by every attribute and a combination of them
+  app.get('/records', async (req, res) => {
+  const allRecords = await Record.find(req.query);
```
Comment (lindahz) [inline server.js:67]: 👏🏻

**Review by lindahz — APPROVED** [review]
Liza, you've done a great job with this project! 😍

I'm impressed and learned many new things by reviewing your code. You have created many different endpoints using different methods, and I can tell that you have studied the MongoDB and Mongoose documentation carefully.

Keep up the good work! 💛

File: server.js [hunk 2]
```diff
@@ -3,36 +3,130 @@ import bodyParser from 'body-parser'
...
+  })
+
+  //Limiting returning data
+app.get('/records20', async (req, res) => {
+  const allRecords = await Record.find(req.query).skip(20).limit(20);
+    res.json(allRecords)
+  })
+
+  //Route to find one specific record by id
+app.get('/records/id/:id', async (req, res) => {
```
Comment (KarolCislowski) [inline server.js:78]: `id` is a default key for a document, so proper endpoint would be  `/records/:id`

**Review by KarolCislowski — APPROVED** [review]
Great Work 🥇

**Pull request closed without merging**